packages/pluggableWidgets/notifications-native/src/Notifications.tsx

```diff
@@ -1,67 +1,129 @@
-import { ActionValue, ValueStatus } from "mendix";
-import { useEffect, useRef, useState } from "react";
 import messaging, { FirebaseMessagingTypes } from "@react-native-firebase/messaging";
+import PushNotification from "react-native-push-notification";
+import { executeAction } from "@mendix/piw-utils-internal";
+import { useCallback, useEffect, useRef, useState } from "react";
+import { ActionValue, ValueStatus } from "mendix";
 import "@react-native-firebase/app";
 
 import { ActionsType, NotificationsProps } from "../typings/NotificationsProps";
-import { executeAction } from "@mendix/piw-utils-internal";
 
-interface NotificationData {
+declare type Option<T> = T | undefined;
```

Comment:
Why `declare`? And can't `Option` simply be imported from `mendix`?

Reply:
I followed an example from Data Grid and didn't think much of it. Changed to use exported type from `mendix`. Thanks!

Reply:
@diego-antonelli FYI

Reply:
@tamelbrun changed